### DISCUSSION TITLE
Anyone done a ballscrew conversion?

### DISCUSSION THREAD

**Discussion by Festivejelly** [kachurovskiy/nanoels#198] [General]
I have a Warco WM180 which is essentially a WM210 with a smaller chuck and bore, but other than that its the same.

Was wondering if anyone has converted their small lathe to use ball screws?

I was considering adding a 16mm ballscrew to the Z and a 10mm to the X.

I must say ever since I started using the Nano ELS h4 I very rarely manually machine any more. I almost always use lathecode instead. And I figured to get a bit more accuracy and longevity swapping to ballscrews might be a good idea.

**Comment by vegetate7** [comment]
Hello! I've converted my 180/300 minilathe to ballscrew Z, and I like it :)
I'd say the ballscrew conversion is not the end of manual operations. This could be as simple as adding flywheel to the end of screw. I'm using much complex design with fixed screw and rotating nut in the apron.

**Reply by kotovasia5120** [reply]
> I'm using much complex design with fixed screw and rotating nut in the apron.

Details?

**Reply by vegetate7** [reply]
> > I'm using much complex design with fixed screw and rotating nut in the apron.
> 
> Details?

The ballnut mounted on sleeve along with pulley and angular gear. Sleeve mounted to apron with 2x angular bearings. Motor and manual flywheel shaft mounted to apron in 3d printed casing. First time I've tryed planetary gearbox on flywheel as speed multiplier, but it was too heavy to rotated, so I've shunted gearbox to 1:1 ratio, and now it is just a bearing case for manual shaft.

![cnc_apron_v9_right_cutout](https://github.com/kachurovskiy/nanoels/assets/57148620/a81d3164-1ade-460e-ac5a-368dc2531863)
![cnc_apron_v9_right](https://github.com/kachurovskiy/nanoels/assets/57148620/35270cc0-1892-4595-8a11-9ea38294ea6b)

![180_300_cnc](https://github.com/kachurovskiy/nanoels/assets/57148620/adfed75b-5d12-4e80-8819-b9119acc4e60)

Now I think all of this can be done much simplier. But, beacause all metal parts were already done, I'll still with this construction.
